### DISCUSSION TITLE
Edit how directories are shown when type `ls`

### DISCUSSION THREAD

**Discussion by ViniciusLagoGehrke** [wezterm/wezterm#5261] [Q&A] [answered]
### What Operating System(s) are you running on?

Windows

### Which Wayland compositor or X11 Window manager(s) are you using?

I'm running on Windows 11 but mostly on WSL2 with Ubuntu 22.04.3

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

Is it possible to edit how directories are displayed when we `ls`?
It currently highlights them (no matter the color scheme I use).

Ideally I could change the font color, remove the highlight (background color) and add a `/` at the end of the directory name. 

I research in docs and discussions for those tags: `color`, `schemes`, `directories`, `list`.

**Comment by bew** [comment, marked as answer]
This is not the terminal that you need to configure to change `ls`'s output format & style, but `ls` itself!

For example to add a / after directories you call `ls -F`. Read about the other flags you can use with ls with `man ls` and `ls --help`

For colors, `ls` is configured using the `LS_COLORS` environment variable, see this tutorial: https://www.howtogeek.com/307899/how-to-change-the-colors-of-directories-and-files-in-the-ls-command/

**Reply by ViniciusLagoGehrke** [reply]
Thank you. Then I can set an alias for that then.
